posix.mak

```diff
@@ -22,6 +22,10 @@
 #
 # make std/somemodule.test => only builds and unittests std.somemodule
 #
+# make TZ_DATABASE_FILE_DIR=pathToDirectoryWithTZDatabaseDirFile => The 
```

Comment:
> How about "path to the TZDatabase directory"?

TZ_DATABASE_FILE_DIR defines the path to the import file only. The path itself is in the file. I am not happy with the naming either but your proposal is not correct.

> Also why is it named _FILE_DIR if it's a directory? Wouldn't TZ_DATABASE_DIR be less confusing? (see comment below about generating this file automatically).

Same as above.
It's the directory in which the TZ_DATABASE_FILE will be found in which the path is defined.
To make the naming more clear the import path and the TZ_DATABASE_DIR version flag can be separated.
The call to make will be more complicated this way because one needs to add IMPORT_PATH=path and something like TZ_DATABASE_DIR=1.
The naming would be much more clear though.

Or keep the current solution and go with TZ_DATABASE_IMPORT_FILE_DIR?